egs/babel_universal_am/s5/local/prepend_lang_id.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
how is this different from  using (for example) utils/copy_data_dir.sh --utt-prefix="LANG" --spk-prefix="LANG" ?

Reply:
I had actually made a note of this in the run.sh. I just hadn't gotten around to replacing it. I hadn't realized that copy_data_dir had that option when I first wrote everything.